File: csharp/Entities/Item.cs
using System;

namespace PhpSales.Entities
{
    public class Item
    {
        public string ItemType { get; set; }
        public string Name { get; set; }
        public double Price { get; set; }
        public double Taxes { get; set; }
        public double TaxedPrice { get; set; }
        public double TaxPercentage { get; set; }
        public int Quantity { get; set; }
        public bool Imported { get; set; }
    }
}

File: src/Entities/Item.php
<?php
namespace src\Entities;
class Item
{
    public string $itemType;
    public string $name;
    public float $price;
    public float $taxes;
    public float $taxedPrice;
    public float $taxPercentage;
    public int $quantity;
    public bool $imported;
}
